### ISSUE TITLE
flexget crashes after upgrade from 3.1.10

### ISSUE THREAD

**Issue by SpaceRat42** [Flexget/Flexget#2584]
### Expected behaviour:

Check of configuration passes with flexget 3.1.12+
(3.1.11 untested)

### Actual behaviour:
Check of configuration fails

### Steps to reproduce:
- Step 1: Perform upgrade from 3.1.10 to 3.1.12+
- Execute "flexget check"

#### Config:
Omitted, will provide, if required.
  
#### Log:
<details>
 <summary>(click to expand)</summary>

```
Traceback (most recent call last):
  File "/usr/local/bin/flexget", line 8, in <module>
    sys.exit(main())
    |   |    -> <function main at 0xb65ca270>
    |   -> <built-in function exit>
    -> <module 'sys' (built-in)>

  File "/usr/local/lib/python3.7/dist-packages/flexget/__init__.py", line 41, in main
    manager.start()
    |       -> <function Manager.start at 0xb5165c00>
    -> <flexget.manager.Manager object at 0xb57278b0>

  File "/usr/local/lib/python3.7/dist-packages/flexget/manager.py", line 359, in start
    self.initialize()
    |    -> <function Manager.initialize at 0xb5165ae0>
    -> <flexget.manager.Manager object at 0xb57278b0>

  File "/usr/local/lib/python3.7/dist-packages/flexget/manager.py", line 231, in initialize
    fire_event('manager.upgrade', self)
    |                             -> <flexget.manager.Manager object at 0xb57278b0>
    -> <function fire_event at 0xb56dc858>

  File "/usr/local/lib/python3.7/dist-packages/flexget/event.py", line 110, in fire_event
    result = event(*args, **kwargs)
             |      |       -> {}
             |      -> (<flexget.manager.Manager object at 0xb57278b0>,)
             -> <Event(name=manager.upgrade,func=upgrade_wrapper,priority=128)>

  File "/usr/local/lib/python3.7/dist-packages/flexget/event.py", line 20, in __call__
    return self.func(*args, **kwargs)
           |    |     |       -> {}
           |    |     -> (<flexget.manager.Manager object at 0xb57278b0>,)
           |    -> <function upgrade.<locals>.upgrade_decorator.<locals>.upgrade_wrapper at 0xb1f233d8>
           -> <Event(name=manager.upgrade,func=upgrade_wrapper,priority=128)>

> File "/usr/local/lib/python3.7/dist-packages/flexget/db_schema.py", line 152, in upgrade_wrapper
    new_ver = upgrade_func(current_ver, session)
              |            |            -> <sqlalchemy.orm.session.ContextSession object at 0xb0e8e7b0>
              |            -> 2
              -> <function upgrade at 0xb1f23348>

  File "/usr/local/lib/python3.7/dist-packages/flexget/plugins/filter/delay.py", line 69, in upgrade
    e = Entry(json.loads(row['json'], decode_datetime=True))
        |     |    |     -> (188216, '{"description": "Category: <a href=\\"http://www.limetorrents.cc/browse-torrents/TV-shows/\\">TV shows</a><br />\\n...
        |     |    -> <function loads at 0xb543d390>
        |     -> <module 'flexget.utils.json' from '/usr/local/lib/python3.7/dist-packages/flexget/utils/json.py'>
        -> <class 'flexget.entry.Entry'>

  File "/usr/local/lib/python3.7/dist-packages/flexget/entry.py", line 77, in __init__
    self.update(*args, **kwargs)
    |    |       |       -> {}
    |    |       -> ({'description': 'Category: <a href="http://www.limetorrents.cc/browse-torrents/TV-shows/">TV shows</a><br />\n\t\tSeeds: 0<b...
    |    -> <function MutableMapping.update at 0xb663f108>
    -> <unprintable Entry object>

  File "/usr/lib/python3.7/_collections_abc.py", line 841, in update
    self[key] = other[key]
    |    |      |     -> 'title'
    |    |      -> {'description': 'Category: <a href="http://www.limetorrents.cc/browse-torrents/TV-shows/">TV shows</a><br />\n\t\tSeeds: 0<br...
    |    -> 'title'
    -> <unprintable Entry object>

  File "/usr/local/lib/python3.7/dist-packages/flexget/entry.py", line 229, in __setitem__
    raise plugin.PluginError('Tried to set title to %r' % value)
          |      |                                        -> datetime.date(2011, 11, 6)
          |      -> <class 'flexget.plugin.PluginError'>
          -> <module 'flexget.plugin' from '/usr/local/lib/python3.7/dist-packages/flexget/plugin.py'>

flexget.plugin.PluginError: Tried to set title to datetime.date(2011, 11, 6)
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/flexget/db_schema.py", line 152, in upgrade_wrapper
    new_ver = upgrade_func(current_ver, session)
  File "/usr/local/lib/python3.7/dist-packages/flexget/plugins/filter/delay.py", line 69, in upgrade
    e = Entry(json.loads(row['json'], decode_datetime=True))
  File "/usr/local/lib/python3.7/dist-packages/flexget/entry.py", line 77, in __init__
    self.update(*args, **kwargs)
  File "/usr/lib/python3.7/_collections_abc.py", line 841, in update
    self[key] = other[key]
  File "/usr/local/lib/python3.7/dist-packages/flexget/entry.py", line 229, in __setitem__
    raise plugin.PluginError('Tried to set title to %r' % value)
flexget.plugin.PluginError: Tried to set title to datetime.date(2011, 11, 6)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/flexget", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/flexget/__init__.py", line 41, in main
    manager.start()
  File "/usr/local/lib/python3.7/dist-packages/flexget/manager.py", line 359, in start
    self.initialize()
  File "/usr/local/lib/python3.7/dist-packages/flexget/manager.py", line 231, in initialize
    fire_event('manager.upgrade', self)
  File "/usr/local/lib/python3.7/dist-packages/flexget/event.py", line 110, in fire_event
    result = event(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/flexget/event.py", line 20, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/flexget/db_schema.py", line 163, in upgrade_wrapper
    manager.shutdown(finish_queue=False)
  File "/usr/local/lib/python3.7/dist-packages/flexget/manager.py", line 1002, in shutdown
    raise RuntimeError('Cannot shutdown manager that was never initialized.')
RuntimeError: Cannot shutdown manager that was never initialized.

```
</details>

### Additional information:

- FlexGet version: 3.1.12+ (3.1.11 untested)
- Python version: 
- Installation method: pip
- Using daemon (yes/no): no
- OS and version: raspbian raspberryPi 4.19.97-v7l+ #1293
- https://dpaste.org/CVjW
:

**Comment by paranoidi** [comment]
Not a full log or detailee explanation what you were doing to get this.

**Comment by SpaceRat42** [comment]
Further investigation showed, that the crash was not caused by a newer flexget version rather by the upgrade process itself.
In the table 'delay' one rows ‘title’ was '2011-11-06', this caused flexget throw an exception in the 

Excerpt from /usr/local/lib/python3.7/dist-packages/flexget/plugin.py

        if key == "title":
            if not isinstance(value, (str, LazyLookup)):         
                raise plugin.PluginError('Tried to set title to %r' % value)`
 
After omitting the instance check , but ignoring this specific title I could successfully upgrade the database. 

The real topic of this issue is: Is '2011-11-06' a valid entry for the 'title' column in the 'delay' table?
If no, this issue should be closed.

**Comment by paranoidi** [comment]
@gazpachoking if I recall correctly you added fix for this?

**Comment by gazpachoking** [comment]
Ahh yes, this should have been fixed by 0d440829a2df837698f6806009320ba5491a032d
